src/Drivers/CmdLine/CmdLine.csproj

```diff
@@ -74,13 +74,15 @@
       <Project>{5E21C722-F9C6-4192-95EC-89FCEF2ACDB7}</Project>
       <Name>Decompiler</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\Environments\SysV\SysV.csproj">
+    <ProjectReference Include="..\..\tools\c2xml\C2Xml.csproj">
```

Comment:
Is this change correct?

Reply:
My mistake (editing the .csproj). Corrected in next submission.